### ISSUE TITLE
Test new Tweetest Parameters with a fresh set of application repositories

### ISSUE THREAD

**Issue by SAY-droid427** [adoptium/aqa-tests#2686]

**Comment by smlambert** [comment]
As a starting point at #172 (we should go through others in the list to see if they are suitable for external_custom, but starting at zookeeper)
- doc: https://cwiki.apache.org/confluence/display/ZOOKEEPER/HowToContribute
  - repo: https://github.com/apache/zookeeper 
  - branch: git checkout -b ZOOKEEPER-1234
  - command: ant test

**Comment by SAY-droid427** [comment]
@smlambert Thank you for pointing out the test! Should I check whether the tests can be run using the changes in https://github.com/adoptium/aqa-tests/pull/2660?

**Comment by smlambert** [comment]
Yes, and in fact what I was hoping is that we can get #2660 merged, so then we can run testing in Grinder_Tweetest, to save you from having to run locally.  However, given many committers are out until Monday (so likely won't be merged until then), if you feel inspired please do so.

We should also review the remaining tests listed in #172 to see if others are appropriate candidates for external_custom and could be added to this issues list for 'testing a fresh set'.

**Comment by SAY-droid427** [comment]
> Yes, and in fact what I was hoping is that we can get #2660 merged, so then we can run testing in Grinder_Tweetest, to save you from having to run locally. However, given many committers are out until Monday (so likely won't be merged until then), if you feel inspired please do so.
> 
> We should also review the remaining tests listed in #172 to see if others (in the list that are 'unchecked') are appropriate candidates for external_custom and could be added to this issues list for 'testing a fresh set'.

Thank you so much @smlambert ! I will try to run the test above and all those listed in the related issue list to check if they can be added to our eligible test lists :)

**Comment by SAY-droid427** [comment]
TEST RESULTS FOR ZOOKEEPER with the following parameters set:
EXTERNAL_CUSTOM_REPO:  https://github.com/apache/zookeeper
EXTERNAL_REPO_BRANCH: master
EXTERNAL_TEST_CMD: ant test

Results for make compile:
![make_compile_zookeeper](https://user-images.githubusercontent.com/59281636/124414514-8a390380-dd70-11eb-9010-dd36081fa255.png)

Results for make external_custom:
![make_external_zoo](https://user-images.githubusercontent.com/59281636/124414564-9c1aa680-dd70-11eb-97bc-5dc554398190.png)

**Comment by SAY-droid427** [comment]
@smlambert I was running the tests and was wondering where could I find the EXTERNAL_TEST_CMD for each test?

**Comment by SAY-droid427** [comment]
I am updating this doc with the list of all the tests with their required parameters for external_custom:
https://docs.google.com/document/d/1XBfUIhw6so6F1YCSiPEBVoMf-q4Pt_aMUSpvD2stS7o/edit?usp=sharing
This will be completed by tomorrow.
